### ISSUE TITLE
AsciiDoctor (or DAPS?) thinks it is okay to ignore includes

### ISSUE THREAD

**Issue by ghost** [openSUSE/daps#509]
From a (successful!) Travis run for doc-caasp:

```
Validating DC-caasp-admin (with geekodoc-1.0.2.1-31.1.noarch)...
Using MAIN file /usr/src/app/adoc/book_admin.adoc
   Creating XML from ASCIIDOC...
/usr/bin/asciidoctor --attribute=imagesdir!  \
          --backend=docbook5 --doctype=book \
          --out-file=/usr/src/app/build/.adoc/caasp-admin.xml /usr/src/app/adoc/book_admin.adoc
asciidoctor: ERROR: book_admin.adoc: line 24: include file not found: /usr/src/app/adoc/common_intro_target_audience_i.adoc
asciidoctor: ERROR: book_admin.adoc: line 26: include file not found: /usr/src/app/adoc/common_intro_available_doc_i.adoc
asciidoctor: ERROR: book_admin.adoc: line 28: include file not found: /usr/src/app/adoc/common_intro_feedback_i.adoc
asciidoctor: ERROR: book_admin.adoc: line 30: include file not found: /usr/src/app/adoc/common_intro_typografie_i.adoc
Successfully created XML file /usr/src/app/build/.adoc/caasp-admin.xml
```

AsciiDoctor can't find the includes but DAPS just keeps going. That should probably not happen.

**Comment by ghost** [comment]
( @r0ckarong might be interested)

**Comment by ghost** [comment]
Okay, so the issue is that as long as AsciiDoctor generates anything _at all_, it uses a return code of `0`. So all those warnings and errors are ignored by DAPS and thus make, the only case that will make DAPS stop is when the input adoc file does not exist.

**Comment by fsundermeyer** [comment]
Fixed with 247de74898
